contracts/SSVNetwork.sol

```diff
@@ -143,6 +147,8 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
             validatorCount: 0,
             fee: fee.shrink()
         });
+        operatorsPKs[hashedPk] = 0x01;
```

Comment:
I personally think we should save the operator id here.  
Maybe we wont use it but we could have this data saved for the same cost as just "0x01" and maybe down the road we might want to know this data and it will already be there at the same cost.